test/verify/check-system-terminal

```diff
@@ -105,6 +105,8 @@ PROMPT_COMMAND='printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/
 
         b.assert_pixels("#terminal .terminal-group", "header", ignore=[".terminal-title"])
         b.assert_pixels("#terminal .xterm-text-layer", "text")
+        # Check scrollbar, see https://github.com/cockpit-project/cockpit/issues/18859
+        b.assert_pixels("#terminal .terminal-body", "scrollbar")
```

Comment:
I think should exclude the text as I'm afraid this will depend on timing and cause a test flake. Thinking more about it this might be a bit to tricky to write a pixel test for.

We could try to ignore `.xterm-cursor-layer` but not sure if that matches

Reply:
Could you please explain what you mean by "timing"?

I agree that it's tricky. Taking a screenshot does not fully test whether dragging the scrollbar with the mouse works.